packages/runtime/test/modules/RequestModule.test.ts

```diff
@@ -510,3 +514,82 @@ describe("Handling the rejection and the revocation of a Relationship by the Req
         expect((await sRuntimeServices.consumption.attributes.getAttributes({})).value).toHaveLength(1);
     });
 });
+
+async function establishPendingRelationshipWithPredefinedRequestFlow(
+    sRuntimeServices: TestRuntimeServices,
+    rRuntimeServices: TestRuntimeServices,
+    existingRelationshipAttributeForFurtherSharing?: LocalAttributeDTO
```

Comment:
this is always defined, right?

Reply:
In our case, we only use the function `establishPendingRelationshipWithPredefinedRequestFlow` with the optional parameter and since we cannot export the function from a test file, it would probably make sense to set the parameter to required, adapt the implementation and only make it optional again if necessary (i.e. if we want to call it in this file without specifying a value for this parameter). Is that what you mean? 🐸

Reply:
resolved in call!